### PR TITLE
implement PVWA clean task

### PR DESCRIPTION
**Pull request by nimrody15**

### CODE REVIEW

File: tasks/pvwa_clean.yml [hunk 1]
```diff
@@ -1,2 +1,46 @@
 ---
 # tasks file for pvwa clean
+
+- name: Clean Windows\\Temp Folder
+  win_find:
+    paths: "C:\\Windows\\Temp"
```
Comment (zivshits) [inline tasks/pvwa_clean.yml]: Please use %TEMP% to support user defined windows temp folder. @avishayil

File: tasks/pvwa_clean.yml [hunk 2]
```diff
@@ -1,2 +1,46 @@
...
+    - "{{ pvwa_installation_path }}\\CredFiles\\gwuser.ini"
+
+- name: clean run history
+  win_regedit:
+    path: 'HKCU:\\Software\\Microsoft\\Windows\\CurrentVersion\\Explorer\\RunMRU'
+    state: absent
+    delete_key: yes
+
+- name: clean event logs
+- win_shell: |
```
Comment (zivshits) [inline tasks/pvwa_clean.yml]: Try to use this module:
https://docs.ansible.com/ansible/2.4/win_eventlog_module.html
@avishayil @yogevh

File: tasks/pvwa_clean.yml [hunk 3]
```diff
@@ -1,2 +1,46 @@
...
+
+- name: clean recycle bin
+- win_shell: |
+    try {
+      $Recycler = (New-Object -ComObject Shell.Application).Namespace(0xa)
+      $Recycler.items() | ForEach-Object { rm $_.path -Force -Recurse }
+    } catch {
+      Write-Output "Error occured: $error"
+      exit 1
+    }
```
Comment (zivshits) [inline tasks/pvwa_clean.yml]: New line at the end of the file

File: tasks/pvwa_clean.yml [hunk 4]
```diff
@@ -1,2 +1,46 @@
...
+- name: Clean cred files
+  win_file:
+    path: "{{item}}"
+    state: absent
+  with_items:
+    - "{{ pvwa_installation_path }}\\CredFiles\\appuser.ini"
+    - "{{ pvwa_installation_path }}\\CredFiles\\gwuser.ini"
+
+- name: clean run history
+  win_regedit:
```
Comment (zivshits) [inline tasks/pvwa_clean.yml]: default key should remain

File: tasks/pvwa_clean.yml (hunk 2 above)
Comment (avishayil) [reply tasks/pvwa_clean.yml]: It's better to not use it, as there is no task to clear all event viewer logs.

File: tasks/pvwa_clean.yml (hunk 4 above)
Comment (avishayil) [reply tasks/pvwa_clean.yml]: Fixed using powershell

File: tasks/pvwa_clean.yml (hunk 3 above)
Comment (avishayil) [reply tasks/pvwa_clean.yml]: Updated.

**Pull request merged**